decidim-surveys/db/migrate/20200609090533_check_legacy_tables.rb

```diff
@@ -112,7 +104,11 @@ def migrate_legacy_data
       end
     end
   end
+
+  def development_app?
```

Comment:
Where is this method used @tramuntanal ?

Reply:
nowhere... I required it in a previous implementation but when reworking the PR I forgot to remove it.  I've just removed it now